### ISSUE TITLE
Swift 4 support

### ISSUE THREAD

**Issue by akirmse** [tid-kijyun/Kanna#159]
### Description:

Could you please migrate the code to Swift 4?  The changes are very simple.

### Installation method:
- [ ] Carthage
- [x ] CocoaPods(1.1.0 or later)
- [ ] Swift Package Manager
- [ ] Manually
- [ ] other: ()

### Library version:
- [ ] v2.1.1
- [x ] other: (2.2.1)

### Xcode version:
- [ ] 8.1 (Swift 3) 
- [ ] 8.1 (Swift 2.3)
- [ ] 7.3.1
- [x ] other: (9)

**Comment by RudyB** [comment]
@akirmse You can use this line of code in your Podfile in meantime

`pod "Kanna", :git => 'https://github.com/tid-kijyun/Kanna.git', :branch => 'feature/v4.0.0'`

or add this to the bottom of your Podfile for your swift 4 project
```
post_install do |installer|
	installer.pods_project.targets.each do |target|
		swift3_projects = ['Kanna']
		if swift3_projects.include? target.name
			target.build_configurations.each do |config|
				config.build_settings['SWIFT_VERSION'] = '3.2'
			end
		end
	end
end

```

**Comment by suncry** [comment]
good，it's done.

**Comment by ruixingchen** [comment]
Xcode9 now understands Swift3 code, just change the Swift version to 3.2 in build setting, it is fine

**Comment by norman784** [comment]
Swift 4 supports branches just add the dependency like:

```swift
// swift-tools-version:4.0

import PackageDescription

let package = Package(
    name: "demo",
    products: [
        .library(name: "demo", targets: ["demo"]),
    ],
    dependencies: [
        .package(url: "https://github.com/tid-kijyun/Kanna.git", .branch("feature/v4.0.0"))
    ],
    targets: [
        .target(name: "demo", dependencies: ["Kanna"]),
    ]
)

```

**Comment by saldous** [comment]
Any update when your Swift 4 support will be out of beta?

**Comment by tid-kijyun** [comment]
I've just released version 4.0.0.
